obspy/io/nordic/core.py

```diff
@@ -637,6 +644,34 @@ def _read_moment_tensors(tagged_lines, event):
     return event
 
 
+def _read_comments(tagged_lines, event):
+    """
+    Read comment lines from s-file
+
+    :param tagged_lines: Lines keyed by line type
+    :type tagged_lines: dict
+    :returns: updated event
+    :rtype: :class:`~obspy.core.event.event.Event`
+    """
+    if '3' not in tagged_lines.keys():
+        return event
+    # Get comment lines
+    com_lines = tagged_lines['3']
+    # can contain SPEC which is read in as spectral information - 
+    # should such lines not be read as comments?
```

Comment:
Probably not? But I don't know where else they could go. There might be somewhere in QuakeML for it?

Reply:
I was considering to stop th code from reading those type-3 lines that contain structured information (like SPEC, XNEAR). But then I thought that's only useful once we have fully implemented a different I/O way to handle this information. For XNEAR, nothing is implemented yet, while for SPEC, I think only reading is supported yet?

That's why I thought it may be good to keep reading and writing all type-3 lines right now, because that allows all information to be transferred from Nordic -> Obspy -> Nordic.